test/CartoMap.js

```diff
@@ -1,28 +1,30 @@
-import { layerVisibility, RESOLUTION_ZOOMLEVEL_ZERO } from '../constants/layer';
-import CartoValidationError, { CartoValidationTypes as cvt } from '../errors/carto-validation-error';
-import { mat4 } from 'gl-matrix';
+// CartoMap depends on gl-matrix, which after imported is globally available as 'mat4'
+// Constants extracted from /carto-vl/src/constants/layer.js
```

Comment:
This comment is not needed.

Reply:
Do you mean the second one? the one on mat4 or both? 😄

Reply:
Currently `carto.layerVisibility` is not available externally, nor it is `RESOLUTION_ZOOMLEVEL_ZERO`, so I'm kind of replicating them just for this utility class.

I'll keep the comments as a reminder of this.